mysql-test/t/dd_upgrade_encrypted.test

```diff
@@ -0,0 +1,95 @@
+# This test contains zipped 5.7 data directory with innodb partitioned tables.
```

Comment:
Probably not "innodb partitioned tables"?

Reply:
A use case is that we have a 5.7 everything-is-encrypted installation and we want to upgrade to 8.0 and maintain the everything-is-encrypted property.

Reply:
Ok, let me think more on this